src/components/NcAppNavigationCaption/NcAppNavigationCaption.vue

```diff
@@ -206,6 +206,7 @@ export default {
 		flex-shrink: 0;
 		// padding to align the name with the icon of app navigation items
 		padding: 0 calc(var(--default-grid-baseline, 4px) * 2) 0 calc(var(--default-grid-baseline, 4px) * 3);
+		margin-top: 0px;
 		margin-bottom: 12px;
```

Comment:
```suggestion
		margin-block: 0px;
```

Maybe even this, but only if used as heading not when used as list element.